ingestion_tools/scripts/gjensen_config.py

```diff
@@ -105,6 +105,9 @@ def to_dataset_config(
         config["tissue"] = dataset["tissue"]
     if cross_reference:
         config["cross_references"] = cross_reference
+        if "dataset_publications" in config["cross_references"]:
```

Comment:
What are your thoughts on moving this clean up here: https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/db79d2e69dc94a4bc6ac12d8583df673f5f5d518/ingestion_tools/scripts/gjensen_config.py#L383?